plugins/ja3_fingerprint/ja3_fingerprint.cc

```diff
@@ -56,8 +67,22 @@ static int             global_modify_incoming_enabled = 0;
 
 struct ja3_data {
   std::string ja3_string;
-  char        md5_string[33];
+  char        md5_string[ja3_hash_hex_string_with_null_terminator_length];
   char        ip_addr[INET6_ADDRSTRLEN];
+
+  char const *
+  update_fingerprint()
+  {
+    // Validate that the buffer is the same size as we will be writing into.
+    static_assert(ja3_hash_hex_string_with_null_terminator_length == sizeof(this->md5_string));
```

Comment:
At this point I am really questioning whether this assertion has any value. We're checking that the length of the buffer is what we declared it as. Could this break in any case other than someone changing the code to not use the constant, or a compiler having a very serious bug?